### PR TITLE
Prevent saving the protocol when it is locked by another user.

### LINKED ISSUE #1564: Protokoll kann trotz fremdem Lock gespeichert werden
Das Protokoll kann gespeichert werden, auch wenn dieses von einem anderen Benutzer gelockt ist.

### PR DESCRIPTION
**Pull request by deiferni**
Fixes #1564.

Remain on the form after a the error occurred, when the other user unlocks the protocol without modifications, it can be saved again by the current user.

screen shot 2016-02-02 at 16 58 53

Also, cleanup changelog.

### CODE REVIEW

**Comment by deiferni** [comment]
@lukasgraf plz have a look

**Comment by deiferni** [comment]
@lukasgraf rebased & updated

**Comment by deiferni** [comment]
@lukasgraf rebased & updated

File: docs/HISTORY.txt [hunk 1]
```diff
@@ -22,19 +25,21 @@ Changelog
...
-  parent, the previous reference number is recycled. [jone]
-
-- Remove "reference_number" from catalog metadata. [jone]
-
+  parent, the previous reference number is recycled.
+  [jone]
 
-4.6.3 (2016-01-25)
-------------------
+- Remove "reference_number" from catalog metadata.
```
Comment (lukasgraf) [inline docs/HISTORY.txt]: Rebase fail?

File: opengever/meeting/browser/meetings/protocol.py [hunk 2]
```diff
@@ -201,11 +203,27 @@ def handleApply(self, action):
         if self.has_write_conflict():
             self._has_write_conflict = True
             return
+        if self.is_locked_by_another_user():
+            self._is_locked_by_another_user = True
+            return
```
Comment (lukasgraf) [inline opengever/meeting/browser/meetings/protocol.py]: There's still a TOCTTOU race condition here - but I don't think we can really get rid of that without digging deep into Plone's internals.

**Comment by lukasgraf** [comment]
Looks good to me :+1:

**Pull request merged**